packages/instrumentor/plugins/functionHooks.test.ts

```diff
@@ -20,17 +20,69 @@
 import { instrumentAndEvalWith } from "./testhelpers";
 import { functionHooks } from "./functionHooks";
 import * as hooking from "@jazzer.js/hooking";
+import { Hook, trackedHooks } from "@jazzer.js/hooking";
 
 const expectInstrumentationEval = instrumentAndEvalWith(
 	functionHooks("pkg/lib/a")
 );
 
 registerHookManagerGlobally();
 
+//eslint-disable-next-line @typescript-eslint/no-explicit-any
+function withDebug(fn: () => void): jest.Mock<any, any> {
```

Comment:
I think, the helper functions would look better at the bottom of the file. WDYT?

Reply:
At the moment, the tests are sandwiched between two blocks of helper function definitions. Our other tests suites define helper functions after the tests. I think, it's good to stay consistent.

Reply:
I’ll pull mine to the end of the file then